src/components/NostoProduct/store.ts

```diff
@@ -66,15 +77,18 @@ export function createStore(element: NostoProduct) {
   function listen<T extends keyof Events>(k: T, cb: Listener<T>) {
     const mapping: Listener<T>[] = listeners[k] || (listeners[k] = [])
     mapping.push(cb)
-    cb(state[k])
+    if (state[k] !== undefined && state[k] !== null) {
```

Comment:
Can this value be `null`? The type specifies it to be undefined but not nullable?